libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/HttpRequestMocks/HttpResponseMockContent.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using static Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.HttpRequestMocks.HttpResponseMock;
```

Comment:
I don't remember what "Using Static" does. Can you add a comment here. 

Consider not using this. The 4 C# programmers doing this code review on the conference call - none of us know what this does.

Reply:
Removed.